vision/roadmap.md

```diff
@@ -69,11 +77,11 @@ Developers will be able to build mobile client apps in Flutter, React Native, an
 - [Read more: Swift](https://github.com/xmtp/xmtp-ios/issues/7)
 - [Read more: Flutter](https://github.com/xmtp/xmtp-flutter/issues/4)
 
-### Message any address
+### Push notifications - example server
 
-Individuals will be able to send encrypted DMs and announcements to addresses that have not yet created XMTP identities. Recipients will be notified of these messages upon first connection to XMTP.
+Apps will be able to leverage our example server to subscribe users for push notifications to their devices upon receiving new messages.
```

Comment:
```suggestion
Apps will be able to leverage an example server to subscribe users for push notifications to their devices upon receiving new messages.
```

Nit; in the context of xmtp.org, "our" is ambiguous as to who the speaker is. Whereas it can be understood to mean XMTP Labs in context of xmtplabs.com.